SWLOR.CLI/EnhancementItemBuilder.cs

```diff
@@ -79,7 +79,9 @@ internal class EnhancementItemBuilder
             { "Shield Recharge Rate", 49 },
             { "Thermal Damage", 51 },
             { "Thermal Defense", 55 },
-            { "Module Bonus", 45 }
```

Comment:
This didn't need to be removed. 45 is still Module Bonus unless I'm missing something?

Reply:
This was almost certainly in error and I can undo this change later tonight when I get home.

The only changes I recall doing here were with accuracy, attack, force attack and agility. I would have had no reason to adjust module bonuses.

Reply:
I got it already, no worries. Thanks